src/main/java/com/epam/edumanagementsystem/student/rest/api/StudentController.java

```diff
@@ -163,20 +159,34 @@ public String editStudentPersonalInformation(@ModelAttribute("existingStudent")
     @Operation(summary = "Adds image to selected student's profile")
     public String addPic(@PathVariable("id") Long id,
                          @RequestParam("picture") MultipartFile multipartFile) {
-        StudentDto studentById = studentService.findById(id);
-        User userByEmail = userService.findByEmail(studentById.getEmail());
-        studentService.addProfilePicture(studentById, multipartFile);
+        logger.debug(multipartFile.getName()+" "+studentService.findById(id));
+        studentService.addProfilePicture(studentService.findById(id), multipartFile);
         return REDIRECT + id + PROFILE;
     }
 
     @GetMapping("/{id}/image/delete")
     @Operation(summary = "Deletes image to selected student's profile")
     public String deletePic(@PathVariable("id") Long id) {
-        StudentDto studentById = studentService.findById(id);
-        imageService.deleteImage(studentById.getPicUrl());
-        studentService.deletePic(studentById.getId());
+        logger.info("user want delete image"+" " +id);
```

Comment:
Use {} after image so as the id would be included in log. And also perephrase the message, write sth about the method or the flow of the process, rather than what user wants.